editor/groups_editor.cpp

```diff
@@ -803,7 +803,7 @@ void GroupsEditor::_bind_methods() {
 void GroupsEditor::_node_removed(Node *p_node) {
 	if (scene_root_node == p_node) {
 		scene_groups_for_caching = scene_groups;
-		callable_mp(this, &GroupsEditor::_cache_scene_groups).call_deferred(p_node);
+		callable_mp(this, &GroupsEditor::_cache_scene_groups).call(p_node);
```

Comment:
```suggestion
		_cache_scene_groups(p_node);
```

Reply:
Yeah, suppose there is no need to even go through callable.